docs/arch/adr-47.md

```diff
@@ -0,0 +1,69 @@
+# ADR 47: Highlighter Task
+
+1 Mar 2023
+
+## Context
+To support OCR verification (the DigiLeap project), with subjects that consist of an image and OCR output as text file, we will create a highlighter task. The highlighter task will allow volunteers to highlight text in the OCR output with a label or labels defined by researchers.
+
+The FEM highlighter task is an iteration on the [PFE highlighter task](https://github.com/zooniverse/Panoptes-Front-End/tree/master/app/classifier/tasks/highlighter).
+
+## Features
+To use the highlighter task a volunteer selects a label from the task area, highlights the related text in the text subject viewer, then selects a different label from the task area to highlight additional text in the text subject viewer or completes the task. Highlighted text can be unhighlighted.
+
+The highlighter task label(s) description and color is defined by researchers.
+
+The following is an example of a highlighter task annotation with labels "Scientific Name", "Collector Name", "Habitat", and "Location":
+```json
+{
+  "value": [
+    {
+      "labelInformation": {
```

Comment:
This annotation example is based on the existing PFE highlighter task annotation. Should this be snake case? `taskType` is camel case, so maybe it's ok this is also camel case.

Reply:
Which is preferred by Caesar?

I’d continue with the existing annotation format, unless there’s a pressing reason to change it. 

The task labels will be localised, so I’d assume that aggregation uses the project primary language.